device_test.go

```diff
@@ -128,6 +122,33 @@ func testNewDevices(t *testing.T) {
 	}
 
 	devices, err := newDevices([]DeviceInfo{deviceInfo})
+	assert.NotNil(t, err)
+
+	format := strconv.FormatInt(major, 10) + ":" + strconv.FormatInt(minor, 10)
+	ueventPathPrefix := filepath.Join(sysDevPrefix, "char", format)
+	ueventPath := filepath.Join(ueventPathPrefix, "uevent")
+
+	// Return true for non-existant /sys/dev path.
```

Comment:
You got a golint error with mispell `non-existant` instead of `non-existent`

Reply:
yes, fixing it